tools/model_converters/stdc2mmseg.py

```diff
@@ -0,0 +1,184 @@
+        '0', '1', '2.0', '2.1', '2.2', '2.3', '3.0', '3.1', '3.2', '3.3',
+        '3.4', '4.0', '4.1', '4.2'
+    ]
+    for k, v in ckpt.items():
+        ori_k = k
+        flag = False
+        if 'cp.' in k:
+            k = k.replace('cp.', '')
+        if 'features.' in k:
+            num_layer = int(k.split('.')[1])
+            feature_key_lst = 'features.' + str(num_layer) + '.'
+            stages_key_lst = 'stages.' + stage_lst[num_layer] + '.'
+            k = k.replace(feature_key_lst, stages_key_lst)
+            flag = True
+        if 'conv_list' in k:
+            k = k.replace('conv_list', 'layers')
+            flag = True
+        if 'avd_layer.' in k:
+            if 'avd_layer.0' in k:
+                k = k.replace('avd_layer.0', 'avg_pool.conv')
+            elif 'avd_layer.1' in k:
+                k = k.replace('avd_layer.1', 'avg_pool.bn')
+            flag = True
+        if 'conv_last' in k:
+            k = k.replace('conv_last', 'final_conv')
+            flag = True
+        if 'arm' in k:
+            if 'arm16' in k:
+                k = k.replace('arm16', 'arms.0')
+            elif 'arm32' in k:
+                k = k.replace('arm32', 'arms.1')
+            flag = True
+        if '_atten' in k:
+            if 'conv_atten' in k:
+                k = k.replace('conv_atten', 'conv_attn.conv')
+            elif 'bn_atten' in k:
+                k = k.replace('bn_atten', 'conv_attn.bn')
+            flag = True
+        if 'conv_head' in k:
+            if 'conv_head32' in k:
+                k = k.replace('conv_head32', 'convs.1')
+            elif 'conv_head16' in k:
+                k = k.replace('conv_head16', 'convs.0')
+            flag = True
+        if 'ffm.conv' in k:
+            if 'ffm.convblk' in k:
+                k = k.replace('ffm.convblk', 'ffm.conv0')
+            elif 'ffm.conv1' in k:
+                k = k.replace('ffm.conv1', 'ffm.conv1.conv')
+            elif 'ffm.conv2' in k:
+                k = k.replace('ffm.conv2', 'ffm.conv2.conv')
+            flag = True
+        if 'conv_out.conv' in k:
+            if 'conv_out.conv.conv.' in k:
+                k = k.replace('conv_out.conv.conv.',
+                              'decode_head.convs.0.conv.')
+            elif 'conv_out.conv.bn.' in k:
+                k = k.replace('conv_out.conv.bn.', 'decode_head.convs.0.bn.')
+            elif 'conv_out.conv_out' in k:
+                k = k.replace('conv_out.conv_out', 'decode_head.conv_seg')
+            flag = True
+        if 'x' in k:
+            flag = False
+        if 'conv_avg' in k:
+            flag = True
+        if flag:
+            new_state_dict[k] = ckpt[ori_k]
+
+    return new_state_dict
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Convert keys in official pretrained STDC1/2 to '
+        'MMSegmentation style.')
+    parser.add_argument('src', help='src model path')
+    # The dst path must be a full path of the new checkpoint.
```

Comment:
Is this comment necessary?

Reply:
I think it's OK. Identical comments also exist [vit2mmseg](https://github.com/open-mmlab/mmsegmentation/blob/91cbe06c8ec30c0f2f4bb2bbeccd363d26a91676/tools/model_converters/vit2mmseg.py#L51), [swin2mmseg](https://github.com/open-mmlab/mmsegmentation/blob/91cbe06c8ec30c0f2f4bb2bbeccd363d26a91676/tools/model_converters/swin2mmseg.py#L70) and [mit2mmseg](https://github.com/open-mmlab/mmsegmentation/blob/91cbe06c8ec30c0f2f4bb2bbeccd363d26a91676/tools/model_converters/mit2mmseg.py#L65).